File: basic/1022.cpp
#include <iostream>
#include <string>
#include <algorithm>
using namespace std;

string toD(int num, int D) {
    string ans = "";
    do {
        ans += (num % D)+'0';
        num /= D;
    } while (num != 0);
    reverse(ans.begin(),ans.end());
    return ans;
}
int main() {
    int A, B, D;
    cin >> A >> B >> D;
    cout << toD(A + B, D) << endl;
    return 0;
}

File: basic/1022.py
def toD(num, D):
    ans = ""
    while 1:
        ans = str(num % D) + ans
        num //= D
        if num==0:
            break
    return ans


(A, B, D) = [int(i)for i in input().split(" ")]
print(toD(A + B, D))
